modules/apple_support/0.11.0/presubmit.yml

```diff
@@ -0,0 +1,20 @@
+matrix:
+  platform: ["centos7", "debian10", "macos", "ubuntu2004", "windows"]
+
+tasks:
+  verify_targets:
+    name: "Build targets under //lib"
+    platform: ${{ platform }}
+    build_targets:
+    - '@apple_support//lib/...'
+
+bcr_test_module:
+  module_path: "test"
+  matrix:
+    platform: ["macos"]
+  tasks:
+    run_test_module:
+      name: "Run test module"
+      platform: ${{ platform }}
+      build_targets:
```

Comment:
should this not be `test_targets` or something? maybe one of the other identifiers is magic instead?

Reply:
Good catch. I copy/pasted from [protobuf](https://github.com/bazelbuild/bazel-central-registry/blob/22e483ee47743eeab422febbb94a3e2bb72948bb/modules/protobuf/3.19.0/presubmit.yml#L22), so I assume that's a bug?

Reply:
Oh, the protobuf examples directory doesn't contain any tests, just build targets, which is also useful to verify if protobuf works correctly. That's why we have `build_targets` here.

Reply:
I changed it to `test_targets` for us.

Reply:
See https://github.com/bazelbuild/apple_support/pull/100#discussion_r787610126, without a WORKSPACE under `test` directory, you're actually running the tests from main module instead of a test module.